habitat-baselines/habitat_baselines/rl/hrl/hl/llm_policy.py

```diff
@@ -96,7 +119,11 @@ def get_next_skill(
 
             # Query the LLM agent with the current observations
             # to get the next action and arguments
-            llm_output = self._llm_agent.chat(observations[batch_idx])
+            llm_output = self.llm_agent.chat(semantic_observation)
+            print("=================llm_output===================")
+            print("Agent: ", self.llm_agent.name)
+            print(llm_output)
+            print("==============================================")
             if llm_output is None:
                 next_skill[batch_idx] = self._skill_name_to_idx["wait"]
                 skill_args_data[batch_idx] = ["1"]
```

Comment:
Maybe we could increase the `wait` step to 5 or 10? The average steps for navigation and pick/place policies would be 100 or less.

Reply:
Hey bro, what do you think about it @SgtVincent